app/src/main/java/crux/bphc/cms/models/SingleLiveEvent.kt

```diff
@@ -0,0 +1,14 @@
+package crux.bphc.cms.models
```

Comment:
This is a utility class right? Might be better putting it on `crux.bphc.cms.utils`.

Reply:
The utilities are classes with common functions that are used throughout the app. SingleLiveEvent makes sure that updated LiveData values are not handled again when the app is reopened for example. For example, for an error event, the Toast/Snackbar should be displayed only once so it checks whether it's been handled or not and only returns the value then. So it's assisting us in modelling the data which is why I kept it in `crux.bphc.cms.models.`